### PR TITLE
overwridden log rotaion config to support size based rotation

### PR DESCRIPTION
**Pull request by anzrmoid**

### CODE REVIEW

File: foxtrot-server/src/main/java/com/flipkart/foxtrot/server/utils/FoxtrotLoggerUtils.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import com.flipkart.foxtrot.server.config.LogRotationFileConfig;
+import com.google.common.base.Optional;
+import com.yammer.dropwizard.config.LoggingConfiguration;
+import com.yammer.dropwizard.logging.LogFormatter;
+
+/**
+ * Created by anzer.moid on 10/02/18.
+ */
+public class FoxtrotLoggerUtils {
+    public static FileAppender<ILoggingEvent> buildFileAppender(LoggingConfiguration.FileConfiguration file,
```
Comment (phantomastray) [inline foxtrot-server/src/main/java/com/flipkart/foxtrot/server/utils/FoxtrotLoggerUtils.java]: @anzrmoid  
Is there any particular reason why we're creating the appenders explicitly ?
Is appender config from .yml not working here ?

Comment (santanusinha) [reply foxtrot-server/src/main/java/com/flipkart/foxtrot/server/utils/FoxtrotLoggerUtils.java]: Agreed. This can be done from logger config.

Comment (anzrmoid) [reply foxtrot-server/src/main/java/com/flipkart/foxtrot/server/utils/FoxtrotLoggerUtils.java]: Foxtrot is on older version of dropwizard 0.6.2 which doesn't support size based configs. hence, explicit creation of appenders.

**Comment by santanusinha** [comment]
Can you move to 0.5.x branch and check if that works for you?

**Comment by anzrmoid** [comment]
@santanusinha no we can't move to that branch as it has es upgrades and many more changes which we don't want. 
@pmohankumar if you like to add anything here

**Pull request merged**